contents/android/IntroToAndroid.md

```diff
@@ -0,0 +1,118 @@
+# Introduction to Android App Development
+
+Authors: [Lee Yan Hwa](https://github.com/leeyh20)
+
+- [Getting Started](#1-getting-started)
+    - [What is Android?](#11-what-is-android)
+    - [Develop your first app today!](#12-develop-your-first-app-today)
+
+- [Understanding Android](#2-getting-started)
+    - [App Components](#22-app-components)
+    - [Activity Lifecycle](#23-activity-lifecycle)
+    - [MVVM and Architecture Components](#24-mvvm-and-architecture-components)
+    - [Resources](#25-resources)
+    - [Testing](#26-testing)
+- [Further Readings](#3-further-readings)
+
+## 1. Getting Started
+
+### 1.1 What is Android?
+Android is a Linux-based mobile phone operating system (OS), created by Android Inc. which was then acquired by Google. It was launched in 2008. Not only does it power phones, it is already the OS of watches, televisions and even car stereo systems.
+
+The source code of Android is made available through the [Android Open Source Project](https://source.android.com/) (AOSP). The original AOSP code is used mainly on the Nexus and Pixel phones that are developed by Google itself. The AOSP code is customized and adapted by original equipment manufacturers (OEMs), such as Samsung and Sony, to run on their respective phones. There is a list of OEMs [here](https://www.android.com/certified/partners/). Android also has many versions, from Android Ice Cream Sandwich to the latest Android Oreo (as of March 2018). The Android ecosystem is quite [fragmented](https://www.androidauthority.com/android-version-distribution-748439/), with mainly 5 Android versions being installed in Android devices today.
+
+The popularity of Android cannot be denied. Google Play Store currently has around 3 million apps as of 2017 ([source](http://www.businessofapps.com/data/app-statistics/)), while Android has become the [world's most popular mobile platform](https://developer.android.com/about/android.html). It's a great time to step into the world of developing Android apps!
+
+### 1.2 Develop your first app today!
```

Comment:
There's another great resource for learning Android. http://web.stanford.edu/class/cs193a/videos.shtml, it's from a Course in Stanford, and it's great for students who have experience with programming(like us) because the pre-req to the class is a course on data structure and algorithms. So the pace is quite good.

Reply:
I have actually linked to this resource below, but I shall make it clearer, thanks!